src/vorta/scheduler.py

```diff
@@ -114,7 +114,7 @@ def create_backup(self, profile_id):
                 logger.error('Error during backup creation.')
         else:
             logger.error('Conditions for backup not met. Aborting.')
-            logger.error(msg['message'])
+            logger.error(msg['message'])  # XXX: in the log, we usually use english, but this might be translated.
```

Comment:
shall we keep this in english, so we can use it for logging?

maybe we could use `translate('message', 'some message') to get it into some specific scope, no matter where the translate calls are.

at one place, there could be a dummy call, just to make it extractable.
then we could hopefully do a late call, like `translate(m)`.

Reply:
decided to do it like that and use a trans_late function, to translate late.